src/sprintThree/Cars/Model.java

```diff
@@ -0,0 +1,47 @@
+package sprintThree.Cars;
+
+//Create methods
+//Set country - should set a country for car object
+//Set model - should set a model for car object
+//Set maker - - should set a maker for car object
+//Get country - Should return country of car
+//Get model - Should return model of car
+//Get maker - - Should return maker of car
+
+class Model extends Car{
```

Comment:
Acccording to the task you have to implement "Car" class and "Main" class. Model is a characteristic of the car(Example of inheritance is: Bus extends Car)